src/script/team/TeamRepository.ts

```diff
@@ -274,12 +307,12 @@ export class TeamRepository {
     const isOtherUser = this.selfUser().id !== userId;
 
     if (isLocalTeam && isOtherUser) {
-      this.userRepository.get_user_by_id(userId).then(userEntity => this._addUserToTeam(userEntity));
-      this.getTeamMember(teamId, userId).then(members => this._updateMemberRoles(members));
+      this.userRepository.get_user_by_id(userId).then(userEntity => this.addUserToTeam(userEntity));
+      this.getTeamMember(teamId, userId).then(member => this.updateMemberRoles(member));
     }
   }
 
-  _onMemberLeave(eventJson) {
+  private onMemberLeave(eventJson: any): void {
```

Comment:
Is it a pure member leave event? If yes, then the type can be taken from the API client.